package-lock.json

```diff
@@ -15699,7 +15699,7 @@
       "dependencies": {
         "jsesc": {
           "version": "0.5.0",
-          "resolved": "https://registry.npmjs.org/jsesc/-/jsesc-0.5.0.tgz",
+          "resolved": "http://registry.npmjs.org/jsesc/-/jsesc-0.5.0.tgz",
```

Comment:
This seems...weird?

Reply:
I occasionally have little things like this change after I run npm install. I think it's fine 🤷‍♂️